specs/HiddenPdfToolbarItems.md

```diff
@@ -0,0 +1,147 @@
+# Background
+When opening a PDF file in Edge, there will be a toolbar at the top. The toolbar provides functionality like print, save and adding annotations.
+This PDF viewer is also available in WebView2. And we provide this new API to enable end developers to customize the PDF toolbar.
+
+
+# Description
+We add a new `HiddenPdfToolbarItems` property in `CoreWebView2Settings`. This API allows end developers to hide buttons on the PDF toolbar. 
+Currently, this API can hide the _save button_, _save as button_, and _print button_.
+By default, `HiddenPdfToolbarItems` is equal to `None` which means no button will be hidden.
+
+# Examples
+## C++
+
+```cpp
+wil::com_ptr<ICoreWebView2> webView;
+void SettingsComponent::ToggleHidePdfToolbarButtons()
+{
+    // Get webView's current settings
+    wil::com_ptr<ICoreWebView2Settings> coreWebView2Settings;
+    CHECK_FAILURE(webView->get_Settings(&coreWebView2Settings));
+    
+    COREWEBVIEW2_PDF_TOOLBAR_ITEM hiddenPdfToolbarItems;
+    CHECK_FAILURE(coreWebView2Settings->get_HiddenPdfToolbarItems(&hiddenPdfToolbarItems));
+    
+    if (hiddenPdfToolbarItems == COREWEBVIEW2_PDF_TOOLBAR_ITEM::COREWEBVIEW2_PDF_TOOLBAR_ITEM_NONE)
+    {
+        CHECK_FAILURE(coreWebView2Settings->put_HiddenPdfToolbarItems(
+            COREWEBVIEW2_PDF_TOOLBAR_ITEM::COREWEBVIEW2_PDF_TOOLBAR_ITEM_PRINT |
+            COREWEBVIEW2_PDF_TOOLBAR_ITEM::COREWEBVIEW2_PDF_TOOLBAR_ITEM_SAVE));
+    }
+    else
+    {
+        CHECK_FAILURE(coreWebView2Settings->put_HiddenPdfToolbarItems(
+            COREWEBVIEW2_PDF_TOOLBAR_ITEM::COREWEBVIEW2_PDF_TOOLBAR_ITEM_NONE));
+    }
+}
+```
+
+## C#
+```c#
+private WebView2 _webView;
+void ToggleHidePdfToolbarButtons(object target, ExecutedRoutedEventArgs e)
+{
+    var coreWebView2Settings = _webView.CoreWebView2.Settings;
+    if(WebViewSettings.HiddenPdfToolbarItems == CoreWebView2PdfToolbarItem.Save | CoreWebView2PdfToolbarItem.Print)
```

Comment:
Enum name should be plural since it's a [flags]: CoreWebView2PdftoolbarItems.

Reply:
Yes good point. Please update the name.